avocado/utils/network.py

```diff
@@ -72,17 +72,13 @@ def find_free_port(start_port=1024, end_port=65535, address="localhost", sequent
 
     :param start_port: header of candidate port range, defaults to 1024
     :param end_port: ender of candidate port range, defaults to 65535
-    :param sequent: Find port sequentially, random order if it's False
     :param address: Socket address to bind or connect
+    :param sequent: Find port sequentially, random order if it's False
     :rtype: int or None if no free port found
     """
-    port_range = range(start_port, end_port)
-    if not sequent:
-        port_range = list(port_range)
-        random.shuffle(port_range)
-    for i in port_range:
-        if is_port_free(i, address):
-            return i
+    port = find_free_ports(start_port, end_port, 1, address, sequent)
```

Comment:
Just as a suggestion: For a better semantic understanding, I would  s/port/ports/.

Besides that, LGTM.

Reply:
But the function is `find_free_port` and `find_free_ports` is called with `count=1`, so I'm expecting 1 port only.

Reply:
I think both PoV are correct, but I tend to agree with the fact that because the return type is a list (even if containing a single item), it deserves to be plural.  Doing `port[0]` calls for explanation IMO, while `ports[0]` does not.  Anyway @willianrampazzo I'm ready to merge this, let me know if you're also OK with the `ports` approach.

Reply:
It doesn't make much sense for me, but I don't have any problem with changing that.

Reply:
OK, I've got the modified version ready to go, so I'm merging it.  Thanks again!

Reply:
> Just as a suggestion: For a better semantic understanding, I would s/port/ports/.
> 
> Besides that, LGTM.

I would suggest keeping it ``port_range`` since ``port`` can generally be used as adjective.